### ISSUE TITLE
Locking accounts on httpxRead Timeout

### ISSUE THREAD

**Issue by codilau** [vladkens/twscrape#40]
Hello, I added this exception in the req function in queue_client.py. I think it makes sense not to lock the accounts in this case.
```
except (httpx.ReadTimeout, httpx.ProxyError):
    retry_count += 1
    if retry_count >= 3:
        logger.warning(f"Httpx error {type(e)}: {e}") 
```
Do you think thats a good ideea @vladkens ?

**Comment by codilau** [comment]
Update. I'm using proxies and this might cause also some errors. To ensure suite possible network lag, i also set timeout=None at the try block.
```
rep = await ctx.clt.request(method, url, params=params, timeout=None)
```

**Comment by vladkens** [comment]
Hi, @codilau.

Fixed problems from first comment in `v0.7.0`.

For timeout configuration create separate ticket please, need to think how to pass timeout param configuration so deep in code calls. Maybe is option to create some global config which can be updated for cases like this.

See changelog: https://github.com/vladkens/twscrape/releases/tag/v0.7.0
